### PR TITLE
Fix `make snapshot`

### PR DESCRIPTION
**Pull request by findepi**
It failed while trying to create :unlabelled tags. See comment in
Makefile for explanation. This logically reverts
6bdb20a6f433e4be72559feb81b16b65b61b7827.

cc @homar

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -124,7 +123,7 @@ snapshot-tags: $(SNAPSHOT_TAGS)
 #
 .PHONY: require-clean-repo require-on-master
 require-clean-repo:
-	test -z "$(shell git status --porcelain)" || ( echo "Repository is not clean"; exit 1 )
+	#test -z "$(shell git status --porcelain)" || ( echo "Repository is not clean"; exit 1 )
```
Comment (homar) [inline Makefile:126]: This should not be commented out imho ;)

**Review by homar — APPROVED** [review]
LG % single comment

File: Makefile (hunk 1 above)
Comment (findepi) [reply Makefile]: thanks! fixed

File: Makefile [hunk 2]
```diff
@@ -161,19 +160,26 @@ $(FLAGDIR)/%.flags: %/Dockerfile $(FLAG_SH)
...
+#     parent of the layer containing the LABEL information, and tagging that as
+#     :unlabelled. This works because all of the LABEL information applied via a
+#     --label option(s) to `docker build' is put in a single layer at the top of
+#     the resulting stack of layers.
+#
+# However
+# - the above no longer worked, because `docker build --label` does not currently
+#   result with a separate layer that could be used for tagging.
+# - the history is retained above as an explanation what the ":unlabelled" supposed
+#   was supposed to me.
```
Comment (losipiuk) [inline Makefile:179]: cannot parse

**Review by losipiuk — APPROVED** [review]
looks fine

File: Makefile (hunk 2 above)
Comment (findepi) [reply Makefile:179]: changed to

```
# - the history is retained above as an explanation what the ":unlabelled" was
#   supposed to mean.
```

**Pull request merged**